react/__mocks__/vtex.styleguide.tsx

```diff
@@ -1,4 +1,10 @@
+// @ts-ignore
```

Comment:
Why some lines need `@ts-ignore` and other don't? Do we actually need `@ts-ignore` here?

Reply:
yes because only some of the components don't have ts typings